server/player/state/unauthenticated_state.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 #
-# Copyright (C) 2018  Maurice van der Pot <griffon26@kfk4ever.com>
+# Copyright (C) 2018 Timo Pomer <timopomer@gmail.com>
```

Comment:
As a rule of thumb, add a copyright statement with your name when you have contributed more than a handful of lines.
But don't remove the existing copyright statements, because they are still applicable.

Reply:
alright. ive never written code with those. just tell me if im messing something up

Reply:
Just readd my copyright and it'll be fine.

Reply:
done